doc/_extensions/vuln_table.py

```diff
@@ -244,12 +243,27 @@ def read_versions(app: Sphinx) -> None:
         logger.error("Could not load version file")
         app.env.nrf_versions = []
 
+def vuln_filter_install(
+    app: Sphinx,
+    pagename: str,
+    templatename: str,
+    context: Dict,
+    doctree: Optional[nodes.Node],
+) -> None:
+    """Install the javascript version filter function."""
+
+    if app.builder.format != "html" or not doctree or pagename != "vulnerabilities":
+        return
+
+    app.add_js_file("js/filter_vuln.js")
```

Comment:
One better way to achieve this (doesn't hardcode pagename):

https://github.com/zephyrproject-rtos/zephyr/pull/41753/commits/2f27a2ae4489738ad19ca6a0780d0abffef72d27#diff-1fbb18b0b4161bcc484f411ff1a1f8bdf0a5b5f789b51cc0859b6e1163c065e0R158-R162